javascript/react-js/router.md

```diff
@@ -84,35 +84,34 @@ ReactDOM.render(
 
-2. Add the `exact` keyword to your routes. So make them look like this:
+2. Add the `exact` keyword to your routes, like so:
 
 ~~~javascript
 <Route exact path="/" component={App} />
 <Route exact path="/profile" component={Profile} />
 
 ~~~
 
-As you can see, this also works. The exact keyword just specifies that the routes path has to match exactly the URL path.
+The exact keyword specifies that the routes path has to match the URL path exactly, as opposed to finding the first character (or string of characters) in the routes list that matches the URL path.
 
-Now you should have enough basics to get started with React routing. There are a lot more features to react-router-dom, which are extremely useful, but out of the scope of this lesson. Just naming two, the history or match object. Definitely go and check out more advanced concepts once you are familiar with the basics.
+You should now have enough basics to get started with React routing. There are a lot more features to react-router-dom which are extremely useful, but out of the scope of this lesson. If you are interested in learning some more, we recommend you look into the history or match object. Definitely go and check out more advanced concepts once you are familiar with the basics.
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
-1. Go and add a few new routes to the application we created above, playing around with it is the best practice. Maybe delete it completely and rewrite it.
-2. Browse a little bit through their [documentation](https://reacttraining.com/react-router/web/guides/quick-start). You don't need to read through all of it, neither understand all of it. Just browse through the concepts we discussed here and re-read them. I'm mentioning it here, because it's a great ressource to refere back to.
+1. Go and add a few new routes to the application we created above; playing around with it is the best practice. Consider deleting it completely and rewriting it for practice.
+2. Browse a little bit through the React Router [documentation](https://reacttraining.com/react-router/web/guides/quick-start). You don't need to read through all of it, nor understand all of it. Just browse through the concepts we discussed here and re-read them. This is a great resource to refer back to.
 3. Watch [this](https://www.youtube.com/watch?v=Law7wfdg_ls&t=850s) video for reviewing. You should be already familiar with those concepts.
 </div>
 
 ### Additional Resources
 
-Have an additional resource that helped you with this lesson?
-
-You can edit the [lesson file](https://github.com/TheOdinProject/curriculum/blob/master/javascript/react-js/router.md) and create a pull request to have one added!
+https://www.youtube.com/watch?v=QUz3k2O3ZJU&ab_channel=TheNetNinja
```

Comment:
Additional resources should be numbered in the markdown with plain English pointing to them. Refer to other lessons for an example on how.